package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "jest": "^29.7.0"
+  },
+  "dependencies": {
+    "find-up": "^7.0.0"
```

Comment:
What do you use `find-up` for in your project? I don't think you need it for `Sprint-1/stretch/aoc-2018-day1/solution.js`.